### ISSUE TITLE
[dependencies] Update django-organizations to 2.0

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/openwisp-users#178]
Switch to django-organizations 2.0, ensuring tests pass the organization admin features work correctly.

**Comment by raisultan** [comment]
hi 👋 @nemesisdesign, I would like to help on this.

**Comment by nemesisdesign** [comment]
@raisultan just go for it :+1:

**Comment by raisultan** [comment]
@nemesisdesign, thanks!

**Comment by robd003** [comment]
I tried upgrading to the master branch where v2.0 is located but I'm getting this error when running tests:

```
<class 'openwisp_users.admin.OrganizationOwnerAdmin'>: (admin.E002) The value of 'raw_id_fields[0]' refers to 'organization_user', which is not an attribute of 'openwisp_users.OrganizationOwner'.
<class 'openwisp_users.admin.OrganizationOwnerAdmin'>: (admin.E002) The value of 'raw_id_fields[1]' refers to 'organization', which is not an attribute of 'openwisp_users.OrganizationOwner'.
<class 'openwisp_users.admin.OrganizationOwnerAdmin'>: (admin.E108) The value of 'list_display[1]' refers to 'organization', which is not a callable, an attribute of 'OrganizationOwnerAdmin', or an attribute or method on 'openwisp_users.OrganizationOwner'.
<class 'openwisp_users.admin.OrganizationOwnerInline'>: (admin.E202) 'openwisp_users.OrganizationOwner' has no ForeignKey to 'openwisp_users.Organization'.
<class 'openwisp_users.admin.OrganizationUserAdmin'>: (admin.E002) The value of 'raw_id_fields[0]' refers to 'user', which is not an attribute of 'openwisp_users.OrganizationUser'.
<class 'openwisp_users.admin.OrganizationUserAdmin'>: (admin.E002) The value of 'raw_id_fields[1]' refers to 'organization', which is not an attribute of 'openwisp_users.OrganizationUser'.
<class 'openwisp_users.admin.OrganizationUserAdmin'>: (admin.E108) The value of 'list_display[0]' refers to 'user', which is not a callable, an attribute of 'OrganizationUserAdmin', or an attribute or method on 'openwisp_users.OrganizationUser'.
<class 'openwisp_users.admin.OrganizationUserAdmin'>: (admin.E108) The value of 'list_display[1]' refers to 'organization', which is not a callable, an attribute of 'OrganizationUserAdmin', or an attribute or method on 'openwisp_users.OrganizationUser'.
<class 'openwisp_users.admin.OrganizationUserInline'>: (admin.E202) 'openwisp_users.OrganizationUser' has no ForeignKey to 'openwisp_users.User'.
openwisp_users.OrganizationUser: (models.E012) 'unique_together' refers to the nonexistent field 'organization'.
openwisp_users.OrganizationUser: (models.E012) 'unique_together' refers to the nonexistent field 'user'.
openwisp_users.OrganizationUser: (models.E015) 'ordering' refers to the nonexistent field, related field, or lookup 'organization'.
openwisp_users.OrganizationUser: (models.E015) 'ordering' refers to the nonexistent field, related field, or lookup 'user'.
```

**Comment by nemesisdesign** [comment]
@robd003 thanks a lot for looking into it, a quick search in django-organizations yielded this issue: https://github.com/bennylope/django-organizations/issues/203 in which other users are reporting the same.

See this comment: https://github.com/bennylope/django-organizations/issues/203#issuecomment-703275858

Maybe you can try that.

**Comment by robd003** [comment]
@nemesisdesign funny enough I was just about to post linking to that. I tried putting the AbstractUser / AbstractOrganization class at the front, but same issue.

**Comment by nemesisdesign** [comment]
@robd003 I see. We may wait until it's resolved or a solution is pointed out. Unless you want to invest more time in it. If you're here for hacktoberfest though, there are many other issues you could help us with. Thanks :+1:

**Comment by robd003** [comment]
@nemesisdesign I just wanted to improve openwisp-users :^)

I'm thinking of using this as part of my SaaS project.

**Comment by nemesisdesign** [comment]
That's great to hear @robd003, here's two issues that would be very useful for everyone:

- https://github.com/openwisp/openwisp-users/issues/170
- https://github.com/openwisp/openwisp-users/issues/171

**Comment by nemesisdesign** [comment]
Closed by #298.
